ml-agents/mlagents/trainers/tests/simple_test_envs.py

```diff
@@ -158,6 +211,7 @@ def _reset_agent(self, name):
         self.step_count[name] = 0
         self.final_rewards[name].append(self.rewards[name])
         self.rewards[name] = 0
+        self.agent_id[name] = (self.agent_id[name] + 1) % 2
```

Comment:
Why not constantly increasing? That's closer to the C# behavior.

Reply:
I can do that. Side note: does C# ever loop around?